Region/BasicStatsCalculator.h

```diff
@@ -31,16 +31,16 @@ template <typename T>
 class BasicStatsCalculator {
     T _min_val, _max_val;
     double _sum, _sum_squares;
-    size_t _num_pixels;
```

Comment:
Nothing wrong with using size_t, since it's just a #define for a 64bit int.

Reply:
got it. max size_t: 18446744073709551615